swarm/tools/runs_gc.py

```
@@ -124,8 +126,13 @@ def get_run_info(run_id: str, run_path: Path, run_type: str) -> RunInfo:
     )
 
 
-def discover_all_runs() -> List[RunInfo]:
-    """Discover all runs from runs/ and examples/ directories."""
+def discover_all_runs(compute_size: bool = True) -> List[RunInfo]:
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)